digitaltwins/samples/device/javascript/sample_device.js

```diff
@@ -157,35 +64,44 @@ const environmentalSensor = new EnvironmentalSensor('environmentalSensor', prope
 const deviceInformation = new DeviceInformation('deviceInformation');
 const modelDefinition = new ModelDefinition('urn_azureiot_ModelDiscovery_ModelDefinition', null, modelDefinitionHandler);
 
-const deviceClient = DeviceClient.fromConnectionString(process.env.DEVICE_CONNECTION_STRING, Mqtt);
-
 const capabilityModel = 'urn:azureiot:samplemodel:1';
```

Comment:
I had hoped that we would deal with this on this go round.  Really want to plumb the code for getting the DCM through on the connect.

Reply:
this is now plumbed in as an option.

---
In reply to: [401188026](https://github.com/Azure/azure-iot-sdk-node/pull/738#discussion_r401188026) [](ancestors = 401188026)